Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM openjdk:latest
-COPY ./target/SemGroupProject-0.1.0.5-jar-with-dependencies.jar /tmp
+COPY ./target/SemGroupProject.jar /tmp
 WORKDIR /tmp
-ENTRYPOINT ["java", "-jar", "SemGroupProject-0.1.0.5-jar-with-dependencies.jar"]
\ No newline at end of file
+ENTRYPOINT ["java", "-jar", "SemGroupProject.jar", "db:3306"]
```

Comment:
the bit we've added at the end here won't work because we haven't did [ticket #105](https://zube.io/craig180885-napier/sem-group_project/c/102)